app/src/main/java/com/example/playlistmaker1/media/ui/MediaFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.example.playlistmaker1.media.ui
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.example.playlistmaker1.R
+import com.example.playlistmaker1.databinding.FragmentMediaBinding
+import com.google.android.material.tabs.TabLayoutMediator
+
+class MediaFragment : Fragment() {
+
+    private lateinit var binding: FragmentMediaBinding
+    private lateinit var tabMediator: TabLayoutMediator
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = FragmentMediaBinding.inflate(inflater,container,false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        binding.viewPager.adapter = MediaViewPagerAdapter(childFragmentManager,lifecycle)
```

Comment:
🍏 MediaViewPagerAdapter можно уже наследовать от другой реализации FragmentStateAdapter, которая принимает в конструкторе родительский фрагмент:
```suggestion
        binding.viewPager.adapter = MediaViewPagerAdapter(this)
```